python/ray/tune/automl/genetic_searcher.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Since this file is not executable, we should omit the `#!` marker. We also shouldn't need the coding flag?

Reply:
done